arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3820,19 +3863,17 @@ arangodb::Result listHotBakupsOnCoordinator(
 arangodb::Result deleteHotBakupsOnCoordinator(
   VPackSlice const payload, VPackBuilder& report) {
 
-  std::vector<std::string> listIds, deleted;
+  std::unordered_map<std::string, BackupMeta> listIds;
```

Comment:
looks unused.

Reply:
Indeed but API design requires that dummy.